syntax = "proto3";

package seller;

service Seller {

    rpc Notify (NotifyRequest) returns (NotifyResponse);
}

message NotifyRequest {
    string message = 1;
}

message NotifyResponse {}